### PR TITLE
SOC-1852 | Re-enable account flow for new auth pages

### PR DESCRIPTION
**Pull request by kvas-damian**
## Links

* https://wikia-inc.atlassian.net/browse/SOC-1852
* https://github.com/Wikia/app/pull/10341

## Description

Redirect user who has requested that my account be closed (but which has not yet been closed) to the "re-enable my account" flow in MediaWiki.

## Reviewers
@bkoval @Bloodmoon24 

Don't merge without https://github.com/Wikia/app/pull/10341
Don't merge until https://github.com/Wikia/pandora/pull/1000 is merged

### CODE REVIEW

**Comment by bkoval** [comment]
LGTM 📫

**Comment by Bloodmoon24** [comment]
LGTM :100:

File: front/auth/app/common/auth-utils.js [hunk 1]
```diff
@@ -1,25 +1,69 @@
...
+
+			let preferences;
+
+			authLogger.info({
+				message: 'Check if account is scheduled to be closed XHR time',
+				value: preferencesRequstEndTime - preferencesRequstStartTime,
+			});
+
+			if (xhr.status === HttpCodes.OK) {
+				preferences = JSON.parse(xhr.responseText);
```
Comment (tomasznapieralski) [inline front/auth/app/common/auth-utils.js]: `preferences` are used only inside this block, you can declare it there

File: server/app/facets/auth/auth-view.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import {disableCache} from '../../lib/caching';
-import {parse} from 'url';
+import * as authUtils from '../../lib/auth-utils';
```
Comment (tomasznapieralski) [inline server/app/facets/auth/auth-view.js]: you are using only one method `getUserPreferencesUrl()`, can you import just it instead all `*`?

**Comment by tomasznapieralski** [comment]
left minor comments, LGTM

**Pull request merged**